habitat_baselines/common/construct_vector_env.py

```diff
@@ -9,6 +9,7 @@
 def construct_envs(
     config: Config,
     workers_ignore_signals: bool = False,
+    is_eval: bool = False,
```

Comment:
I am against this. 
Environments should not know about wether or no they are used for evaluation. Adding this new argument does not change the behavior of the environments and only affects wether a log message or an error happens. If you really want it, rename to `enforce_less_scenes_than_environemnts`.

Reply:
Sure, I'm fine with the name change.

We do want this change. If people have more environments than scenes that will silently cause evaluation to be incorrect.